### ISSUE TITLE
rempl 1.0.5 - ENOENT: no such file or directory, uv_chdir

### ISSUE THREAD

**Issue by Tom910** [rempl/rempl-cli#1]
In rempl 1.0.5 error when you run
```
Error: ENOENT: no such file or directory, uv_chdir
    at Command.action_ (/usr/local/lib/node_modules/rempl-cli/lib/cli.js:18:17)
    at Command.run (/usr/local/lib/node_modules/rempl-cli/node_modules/clap/index.js:749:22)
    at Object.<anonymous> (/usr/local/lib/node_modules/rempl-cli/bin/rempl:10:9)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
    at Module.runMain (module.js:605:10)
    at run (bootstrap_node.js:422:7)
```

**Comment by Tom910** [comment]
performing code `process.chdir(path.resolve(path.dirname(require.resolve('rempl/server')), 'client/dist'));` - get - `/usr/local/lib/node_modules/rempl-cli/node_modules/rempl/server/client/dist`

```
/usr/local/lib/node_modules/rempl-cli/node_modules/rempl/server/client ⌚ 15:55:39
$ ls -l
total 48
-rw-r--r--  1 a.marchenko  admin   400 25 янв 01:28 app.js
-rw-r--r--  1 a.marchenko  admin   315 25 янв 01:33 build-symlink.js
-rw-r--r--  1 a.marchenko  admin   175  3 мар 15:31 build.config
-rw-r--r--  1 a.marchenko  admin   355 25 янв 01:31 index.html
-rw-r--r--  1 a.marchenko  admin  1386 10 фев 00:26 transport.js
drwxr-xr-x  4 a.marchenko  admin   136  3 мар 15:44 type
-rw-r--r--  1 a.marchenko  admin   150 10 фев 00:21 type.js
drwxr-xr-x  6 a.marchenko  admin   204  3 мар 15:44 ui
```

**Comment by lahmatiy** [comment]
@Tom910 Yep, fixing it. Thank you

**Comment by lahmatiy** [comment]
Some files of `rempl` didn't published on npm. Problem fixed, try out `1.0.6`.

**Comment by Tom910** [comment]
thanks It works
